fuzzy_file_nav.py

```diff
@@ -1173,16 +1177,23 @@ def get_files(self, cwd):
                     documents.append(f)
                 else:
                     folders.append(f + ("\\" if PLATFORM == "windows" else "/"))
-        options = sorted(
-            folders + documents,
-            key=lambda f: self.get_key(cwd, f),
-            reverse=True
-        )
+
+        if sublime.load_settings(FUZZY_SETTINGS).get("sort_entries_by_last_accessed", False):
+            options = sorted(
```

Comment:
This is still intermingling the sort of documents and directories. If you want to for this, it needs to be under a setting. Directories should come first followed by files second by default.

Reply:
I'm not even sure if last accessed directories is even that helpful. Sublime may access folders all the time with indexing...I wonder if directories should be ignored in this last accessed sort...

Reply:
I've added a separate setting to control mixing files and folders. I personally like this behavior since a lot of the time, the last accessed file is the one I want and I can simply press enter to open it.
I think the last access time of files and folders is platform specific as well, but having them behind settings should let people decide if their system's behavior is what they want or not.

Reply:
I don't think folders even update access time when you open them. And they don't seem to update just because I access a file under them. I'm wondering if we should just exclude folders from this. It doesn't seem to do what you think it does, at least not on mac.

Reply:
On Ubuntu the folder access time is getting updated when I access a file inside it. In my short testing at least it's behaving how I want it to

Reply:
That won't always be the case: https://www.tiger-computing.co.uk/file-access-time-atime/#:~:text=The%20atime%20is%20updated%20every,data%20was%20cached%20in%20memory..

But let's say it is. Sublime indexes files and folders at different times. This will make these less relevant at times...

If the file is already open in sublime, the file is not re-read, so I think it won't be updated in that case. I guess it will only be relative to when you first opened the file or when you modified it...I guess that is fine.

I do fear indexing will make a mess of this feature, and while not as frequent in updates as accessed times, I wonder if modified times makes more sense.

Reply:
That makes sense! I wondered the same thing too. I added an option to choose between the two last accessed or last modified